python/cudf/cudf/io/parquet.py

```diff
@@ -953,3 +844,24 @@ def __enter__(self):
 
     def __exit__(self, *args):
         self.close()
+
+
+def _default_open_file_options(open_file_options, columns, row_groups):
+    # Copy and update `open_file_options` to include
+    # column and row-group information under the
+    # "precache_options" key. By default, we set
+    # "method" to "parquet", but precaching will
+    # be disabled if the user chooses `method=None`
```

Comment:
would just use a standard docstring here

Reply:
I vaguely remember someone suggesting that private functions should not include a docstring (might have been @shwina ?).  However, your suggestion seems reasonable to me, and I may be remembering incorrectly.

Reply:
I coverted this to a standard(ish) docstring for now

Reply:
I think @shwina has indeed indicated a preference for that previously, but I also don't think we've been particularly consistent in enforcing that one way or the other. I'd say stick with whatever you have for now and we can address this across the code base at some nebulous later date :)